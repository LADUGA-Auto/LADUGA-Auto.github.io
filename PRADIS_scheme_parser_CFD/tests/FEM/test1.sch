<Qucs Schematic 4.4.0>
<Properties>
  <View=0,0,800,800,1,0,0>
  <Grid=10,10,1>
  <DataSet=test1F.dat>
  <DataDisplay=test1F.dpl>
  <OpenDisplay=1>
  <showFrame=0>
  <FrameText0=\x041D\x0430\x0437\x0432\x0430\x043D\x0438\x0435>
  <FrameText1=\x0427\x0435\x0440\x0442\x0438\x043B:>
  <FrameText2=\x0414\x0430\x0442\x0430:>
  <FrameText3=\x0412\x0435\x0440\x0441\x0438\x044F:>
</Properties>
<Symbol>
</Symbol>
<Components>
  <CoSimulation.SolidDomain SolidDomain1 1 290 220 -71 29 0 0 "Сталь" 0 "0" 0 "25" 0 "10.1" 0>
  <CoSimulation.Port Port1 1 190 120 -41 29 0 0 "первый порт" 0 "ThermalFluid" 0>
  <CoSimulation.Port Port2 1 290 120 -41 29 0 0 "второй порт" 0 "Thermal" 0>
  <CoSimulation.Port Port3 1 390 120 -41 29 0 0 "третий порт" 0 "XYZ" 0>
  <CoSimulation.Solver Solver1 1 290 310 -51 29 0 0 "90" 0 "5" 0 "0.005" 0 "jacobian" 0>
  <CoSimulation.Task Task1 1 290 400 -41 29 0 0 "resultFile" 0 "resultDesc" 0>
</Components>
<Wires>
</Wires>
<Diagrams>
</Diagrams>
<Paintings>
</Paintings>
