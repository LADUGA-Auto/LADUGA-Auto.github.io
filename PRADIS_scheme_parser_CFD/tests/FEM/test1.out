[ Module ]      |      [ Name ]       |      [ Active ]       |      [ Properties ]
[ CoSimulation.SolidDomain ] | [ SolidDomain1 ] | [ true ] | [ Solid : Сталь : string ]    [ Velocity : 0 : integer ]    [ Temperature : 25 : integer ]    [ RotationalVelocity : 10 : double ]    

[ CoSimulation.Port ] | [ Port1 ] | [ true ] | [ Name : первый порт : string ]    [ Interface : ThermalFluid : string ]    

[ CoSimulation.Port ] | [ Port2 ] | [ true ] | [ Name : второй порт : string ]    [ Interface : Thermal : string ]    

[ CoSimulation.Port ] | [ Port3 ] | [ true ] | [ Name : третий порт : string ]    [ Interface : XYZ : string ]    

[ CoSimulation.Solver ] | [ Solver1 ] | [ true ] | [ EndTime : 90 : integer ]    [ MaximalIterationCount : 5 : integer ]    [ Tolerance : 0 : double ]    [ Flag.GetJacobian : jacobian : string ]    

[ CoSimulation.Task ] | [ Task1 ] | [ true ] | [ File : resultFile : string ]    [ Description : resultDesc : string ]    

